packages/amazonq/test/e2e_new/amazonq/utils/authUtils.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Workbench, By, WebviewView } from 'vscode-extension-tester'
+import { findItemByText, sleep, waitForElements } from './generalUtils'
+import { testContext } from './testContext'
+
+/* Completes the entire Amazon Q login flow
+
+Currently, the function will
+1. Open AmazonQ
+2. Clicks Company Account
+3. Inputs the Start URL
+4. IMPORTANT: you must click manually open yourself when the popup window asks to open the browser and complete the authentication in the browser** 
+
+TO-DO: Currently this signInToAmazonQ is not fully autonomous as we ran into a blocker when the browser window pops up */
+export async function signInToAmazonQ(): Promise<void> {
+    const workbench = new Workbench()
+    await workbench.executeCommand('Amazon Q: Open Chat')
+
+    await sleep(5000)
+    let webviewView = new WebviewView()
+    await webviewView.switchToFrame()
+
+    const selectableItems = await waitForElements(webviewView, By.css('.selectable-item'))
+    if (selectableItems.length === 0) {
+        throw new Error('No selectable login options found')
+    }
+
+    const companyItem = await findItemByText(selectableItems, 'Company account')
```

Comment:
Why not do all of these without the varaiable assignment. Its making it hard to read.

Reply:
is this what you're suggesting?
```suggestion
    await findItemByText(selectableItems, 'Company account').click();
```

Reply:
We had an internal discussion. Spaces and comments should do it.